config/settings/production.py

```diff
@@ -13,9 +13,10 @@
 DATABASES["default"] = env.db("DATABASE_URL")  # noqa F405
 DATABASES["default"]["ATOMIC_REQUESTS"] = True  # noqa F405
 DATABASES["default"]["CONN_MAX_AGE"] = env.int("CONN_MAX_AGE", default=60)  # noqa F405
-if env("POSTGRES_SSL"):
+if env.bool("POSTGRES_SSL"):
```

Comment:
I think we should be better about using default values when we query the environment. This, for instance, will generate the following if `POSTGRES_SSL` is not set, where it should probably just default to False:

```
    raise ImproperlyConfigured(error_msg) from exc
django.core.exceptions.ImproperlyConfigured: Set the POSTGRES_SSL environment variable
```

Reply:
Makes sense, though I'll set it to default to True if there is no reason not to, that way people have to explicitly disable SSL.